docs/poster.tex

```diff
@@ -35,7 +35,7 @@
         and frustration.
 
         The following example of documentation drift is taken from
-        a convenience method in the project \textit{darglint}.  The parameter
+        a convenience method in the project \textit{darglint2}.  The parameter
```

Comment:
@terrencepreilly I mechanically modified the poster as well. But what would you actually prefer me to do with it?
- drop it from the repository?
- leave it as is, referring to `darglint` instead of `darglint2` everywhere?
- just update the command lines to use `darglint2`, otherwise leave unmodified?
- update everything to `darglint2` as done here?

Reply:
I don't really have an opinion; whatever you want to do with it.  It's under the same MIT license as everything else.